src/modules/config.js

```diff
@@ -9,6 +9,11 @@ const Config = () => {
     enablePlugin: true,
     enableDimMode: false,
 
+    convertTokenValues: false,
+    usdExchangeRate: 1.00,
```

Comment:
please put this in `constants.js`

Reply:
Not sure about moving those values to constants.js they need to be dynamically changed based on what the users saved.

Reply:
ah i see